engine/jobmaster/dm/api.go

```diff
@@ -46,6 +46,23 @@ type JobStatus struct {
 	FinishedUnitStatus map[string][]*metadata.FinishedTaskStatus `json:"finished_unit_status,omitempty"`
 }
 
+// ShardTable represents create table statements of a source table.
+type ShardTable struct {
+	Current string
+	Pending string
+}
```

Comment:
When I read it I can't make sure `Pending` is a DDL or CREATE TABLE. Maybe `Current` and `Next`.
And the type name `ShardTable` can be better reflect it's a chaning, has the previous and post data, not a static table data. If the type name is showed in API we may ask PM's opinion